lib/eventApi.js

```diff
@@ -12,7 +12,7 @@ module.exports = function (app) {
 
     app.get('/events/:name', function (req, res) {
         eventDb.getEventsFilterByName(req.params.name).then(function (events) {
-            if (events) {
+            if (events.length > 0) {
```

Comment:
Was there no reason for the original null check? This looks like it could easily fail with `cannot read property 'length' of null`.